bwi_logging/launch/record.launch

```diff
@@ -1,15 +1,26 @@
 <launch>
 
-  <arg name="directory" default="" />
-  <arg name="topics" default="odom_1hz amcl_pose /diagnostics" />
+  <arg name="directory"    default="" />
+  <arg name="topics"       default="odom_1hz amcl_pose /diagnostics" />
+  <arg name="extra-topics" default="false" />
+  <arg name="extra-prefix" default="extra" />
```

Comment:
Please use an underscore (``_``) in place of the dash (``-``) for these two argument names, as you did below. That is less confusing when accessing the parameters or arguments in C++ or Python programs.